numbers/stat.go

```diff
@@ -48,3 +56,29 @@ func Pearson(a []float64, b []float64) float64 {
 	}
 	return num / (math.Sqrt(sumA) * math.Sqrt(sumB))
 }
+
+/*
```

Comment:
Is this code block old and now commented out for some reason, or is it new code that are notes?

Reply:
this was something that I was thinking about writing to add stats to the program. I think i am going into a different direction, so i will delete